src/gds-upgrade/sass/components/_notification-banner.scss

```diff
@@ -0,0 +1,10 @@
+.govuk-notification-banner {
+  & .govuk-notification-banner__header {
+    .close-button {
+      color:  #fff;
```

Comment:
We should try to use the govuk functions and colours here:
`  @include govuk-font(14, $weight: bold); // Not sure if that font size will work but give it a go, you have 14, 16, 19, 24...
  color: govuk-colour("white");
`